Detectors/CPV/workflow/src/cpv-reco-workflow.cxx

```diff
@@ -62,5 +64,7 @@ o2::framework::WorkflowSpec defineDataProcessing(o2::framework::ConfigContext co
                                              !cfgc.options().get<bool>("disable-mc"),       //
                                              cfgc.options().get<std::string>("input-type"), //
                                              cfgc.options().get<std::string>("output-type") //
+ //                                            cfgc.options().get<bool>("pedestal"),
+ //                                            cfgc.options().get<std::string>("ccdb-url"),
```

Comment:
why do you need to add options which are not needed?

Reply:
hello @shahor02 
they are not needed. I removed them.

Reply:
Dear @shahor02 ,
I'll try to run full system test and let you know the result.
I also want to be sure that protection against missing TFs is working.
We can simulate missing TFs by command 
`o2-raw-file-reader-workflow --input-conf raw/CPV/CPVraw.cfg --loop 100 --drop-tf "CPV,3"`
But what I need to do next to see if everything is working as expected?

Thank you & regards,
Sergey

Reply:
Hi @sevdokim 
you should see in every 3d TF the message you put: `"Sending empty output due to data type input with 0xDEADBEEF"`

Reply:
Dear @shahor02 ,

I tried to run FST using following environment on ubuntu20.04:
`alienv enter O2/latest AEGIS/latest`
Unfortunately the test was failed at 2nd task:
```
Launching task: o2-sim --seed -1 -n 10 --configKeyValues Diamond.width[2]=6. -g pythia8hi -e TGeant4 -j 4 &> sim.log &
command o2-sim --seed -1 -n 10 --configKeyValues Diamond.width[2]=6. -g pythia8hi -e TGeant4 -j 4 had nonzero exit code 1

```
From the sim.log I see following:
`Spawning particle server on PID 1270555; Redirect output to o2sim_serverlog`
`[INFO] Process 1270555 EXITED WITH CODE 0 SIGNALED 1 SIGNAL 6`
from the o2sim_serverlog:
` PYTHIA Error in BeamRemnants::setKinematics: kinematics construction failed  `
`[ESC[01;36m17:11:29ESC[0m][ESC[01;33mWARNESC[0m] Particle to be tracked is not defined in PDG: pdg = 1000200489`
`[ESC[01;36m17:11:29ESC[0m][ESC[01;33mWARNESC[0m] Particle to be tracked is not defined in PDG: pdg = 1000180489`
I suppose the error is not related to CPV but to non-recognition of some ions by the VMC?
I bet that my setup is not well defined and it was a cause of this failure. Any tips how to proceed?

Thank you!
Sergey

Reply:
Hi @sevdokim 
Will run myself to check, we recently switched to G4, might be related to this.

Reply:
hi @shahor02 ,
done from my side! thanks!